src/lib/bootloader/mbr_update.rb

```diff
@@ -181,20 +198,14 @@ def extended_partition_num(disk)
     #  containing device (eg. "/dev/hda4"), disk (eg. "/dev/hda") and
     #  partition number (eg. 4)
```

Comment:
Since we're fixing this method, let's make sure the API docs are OK:
```rb
# @return a Hash { "mbr" => String, "num" => Integer }
# containing device (eg. "/dev/hda4"), disk (eg. "/dev/hda") and
#  partition number (eg. 4)
```
But I do not understand the "device" part. Like, a *master* BR can be in the BR of a *partition*? (/dev/hda4)

Also, the method sometimes returns {}. Does that mean "no partition should be activated"?